app/helpers/admin_view_hours_helper.rb

```diff
@@ -36,4 +36,52 @@ def synthesize_end_date(params)
     Date.civil(params[:end_time][:year].to_i, params[:end_time][:month].to_i,
                params[:end_time][:day].to_i)
   end
+
+  ###############################################################################
+
+  def tutor_hours
+    today = Time.zone.today
+    if today.month < 6
+      start_date = Date.parse("Jan 1 #{today.year}")
+      end_date = Date.parse("Jun 1 #{today.year}")
+    else
+      start_date = Date.parse("Aug 1 #{today.year}")
+      end_date = Date.parse("Dec 31 #{today.year}")
```

Comment:
can a tutor view hours from 2 semesters ago, or should we make it clear that an admin should export data after each semester?

Reply:
My thought process was that a tutor really only cares about how much they've tutored this semester, so that they know if they're on-track to reach a quota or something. If they really needed their info from previous semesters, they could just ask an admin to give it to them, since I can't imagine this would be a very common occurrence.

I could potentially try to let them pick a semester, but it would take quite a bit more work, so it would only really be worth it if we think it's something they might seriously need.

Reply:
as long as there is a way for this data to be seen later on, even if it means an admin needs to export once a semester, I think this is sufficient.